types/libp2p/index.d.ts

```diff
@@ -84,6 +84,8 @@ declare namespace LibP2p {
         remotePeer: import("peer-id")
     }
 
+    export type PeerType = Peer;
```

Comment:
Can we just export `Peer`?

Reply:
circular reference, no idea how to fix other than namespace all those classes

Reply:
```typescript
declare namespace LibP2p {
  ...
  export interface Peer {
    ...
  }
}

// below

readonly peers: Map<string, LibP2p.Peer>;
```